src/main/java/org/fogbowcloud/manager/core/models/orders/OrderState.java

```diff
@@ -1,10 +1,33 @@
 package org.fogbowcloud.manager.core.models.orders;
 
+import com.fasterxml.jackson.annotation.JsonValue;
+
 public enum OrderState {
-    OPEN,
-    PENDING,
-    SPAWNING,
-    FULFILLED,
-    FAILED,
-    CLOSED;
+    OPEN("open"),
+    PENDING("pending"),
+    SPAWNING("spawning"),
+    FULFILLED("fulfilled"),
+    FAILED("failed"),
+    CLOSED("closed");
+
+    private String value;
+
+    OrderState(String value) {
+        this.value = value;
+    }
+
+    @JsonValue
+    public String getValue() {
+        return value;
+    }
+
```

Comment:
doesn't enum already has something like this fromValue method?

Reply:
It doesn't.

Reply:
how about the OrderState.valueOf? Doesn't this do that?

Reply:
Fixed! I can't use this before because the Enum had no value. Thanks!